### DISCUSSION TITLE
Order by for Table

### DISCUSSION THREAD

**Discussion by praj18** [chakra-ui/chakra-ui#5245] [Q&A] [answered]
I have several columns which are composed of different integer data, and I want to be able to order them dynamically.

Is there a way I can order the rows of a table by clicking on the column header?

**Comment by primos63** [comment, marked as answer]
You may want to check out [Chakra UI with React Table](https://chakra-ui.com/guides/integrations/with-react-table#usage)

**Reply by praj18** [reply]
Thank you so much. I saw this a couple of months ago but I couldn't find it again, so I thought I was imagining stuff.

**Reply by jackherizsmith** [reply]
for anyone trying the old link, the new one is https://v2.chakra-ui.com/getting-started/with-react-table
